### DISCUSSION TITLE
Problem installing Homebrew on Mac M1

### DISCUSSION THREAD

**Discussion by LogicKahanHai** [Homebrew/discussions#4059] [Getting started] [answered]
I tried to install Homebrew by running the following command in my terminal:- 

`/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`

but it gives this error every time... Please help me... 
Error

`curl: (28) Failed to connect to raw.githubusercontent.com port 443 after 75231 ms: Operation timed out` 

tried to look for solutions everywhere.. but I am unable to find any.. please guide me as I am new to this...

My Mac version: macOS Ventura 13.1

**Comment by p-linnane** [comment, marked as answer]
It's something with the connection between your machine and GitHub, nothing Homebrew specific.

See #666.

**Reply by LogicKahanHai** [reply]
Yes thanks a lot... I did some digging after this... Turns out it was a problem with my ISP.. worked when tried with another... Thanks for the help though

**Reply by starlightknown** [reply]
hey, could you tell me how you solved it? I'm facing the same issue and nothing worked

**Reply by LogicKahanHai** [reply]
So if you're in India and you use data services from JIO you'll face issues downloading from "raw.githubusercontent.com" (downloading homebrew requires call to this website) 
Try another Internet provider and it should be good

**Reply by kanthgithub** [reply]
@LogicKahanHai yeah, on JIO its giving same error

**Reply by kanthgithub** [reply]
this worked as mentioned in #666
even being in JIO

```
% git config --global http.postBuffer 500M
% git config --global http.maxRequestBuffer 100M
% git config --global core.compression 0
```

**Reply by Anyone77** [reply]
use vpn bro , it will be done !

**Reply by keyurCh** [reply]
yes i checked with gtpl broadband its not worked but when try with vodafone mobile data its work
